hathor/transaction/transaction.py

```diff
@@ -272,6 +273,16 @@ def verify_basic(self, skip_block_weight_verification: bool = False) -> None:
         self.verify_weight()
         self.verify_without_storage()
 
+    def verify_checkpoint(self, checkpoints: List[Checkpoint]) -> None:
+        assert self.storage is not None
+        meta = self.get_metadata()
+        # at least one child must be checkpoint validated
+        for child_tx in map(self.storage.get_transaction, meta.children):
+            if child_tx.get_metadata().validation.is_checkpoint:
```

Comment:
Maybe rename `is_checkpoint` as I pointed out before.

Reply:
Maybe it should be a method call (instead of a `@property`).

Reply:
Fixed.

Reply:
Fixed in #292 .